lib/console.js

```diff
@@ -166,6 +170,36 @@ Console.prototype.assert = function assert(expression, ...args) {
   }
 };
 
+// Defined by: https://console.spec.whatwg.org/#clear
+Console.prototype.clear = function clear() {
+  // It only makes sense to clear if _stdout is a TTY.
+  // Otherwise, do nothing.
+  if (this._stdout.isTTY) {
+    const { cursorTo, clearScreenDown } = require('readline');
+    cursorTo(this._stdout, 0, 0);
+    clearScreenDown(this._stdout);
+  }
+};
+
+// Defined by: https://console.spec.whatwg.org/#count
+Console.prototype.count = function count(label = kDefaultCount) {
```

Comment:
Given that, I've no problem with using `default` as the label.

@TimothyGu ... I'm curious why you want `this[kCounts]` to be a map. It would not have an impact on the ability to use `Symbol` as a key given the `toString()` restriction that would still exist. Using the dictionary mode object works just as well. Can you elaborate?